user_guide_src/source/database/configuration.rst

```diff
@@ -108,49 +108,50 @@ default group's configuration settings. The values should be name following this
+                 You can set a fully qualified classname to use your custom driver.
+**DBPrefix**     An optional table prefix which will added to the table name when running
+                 :doc:`Query Builder <query_builder>` queries. This permits multiple CodeIgniter
+                 installations to share one database.
+**pConnect**     true/false (boolean) - Whether to use a persistent connection.
+**DBDebug**      true/false (boolean) - Whether to throw exceptions or not when database errors occur.
+**charset**      The character set used in communicating with the database.
+**DBCollat**     The character collation used in communicating with the database (``MySQLi`` only).
+**swapPre**      A default table prefix that should be swapped with ``DBPrefix``. This is useful for distributed
+                 applications where you might run manually written queries, and need the prefix to still be
+                 customizable by the end user.
+**schema**       The database schema, default value varies by driver. (Used by ``Postgres`` and ``SQLSRV``.)
+**encrypt**      Whether or not to use an encrypted connection.
+                 ``SQLSRV`` driver accepts true/false
+                 ``MySQLi`` driver accepts an array with the following options:
+                 * ``ssl_key``    - Path to the private key file
+                 * ``ssl_cert``   - Path to the public key certificate file
+                 * ``ssl_ca``     - Path to the certificate authority file
+                 * ``ssl_capath`` - Path to a directory containing trusted CA certificates in PEM format
+                 * ``ssl_cipher`` - List of *allowed* ciphers to be used for the encryption, separated by colons (``:``)
+                 * ``ssl_verify`` - true/false; Whether to verify the server certificate or not (``MySQLi`` only)
+**compress**     Whether or not to use client compression (``MySQLi`` only).
+**strictOn**     true/false (boolean) - Whether to force "Strict Mode" connections, good for ensuring strict SQL
+                 while developing an application (``MySQLi`` only).
+**port**         The database port number.
+**foreignKeys**  true/false (boolean) - Whether or not to enable Foreign Key constraint (``SQLite3`` only).
+
+                 .. important:: SQLite3 Foreign Key constraint is disabled by default.
+                     See `SQLite documentation <https://www.sqlite.org/pragma.html#pragma_foreign_keys>`_.
+                     To enforce Foreign Key constraint, set this config item to true.
+**busyTimeout**  milliseconds (int) - Sleeps for a specified amount of time when a table is locked (``SQLite3`` only).
+**numberNative** true/false (boolean) - Whether or not to enable MYSQLI_OPT_INT_AND_FLOAT_NATIVE (``MySQLi`` only).
+===============  ===========================================================================================================
```

Comment:
This table is broken. `**numberNative**` (the longest item) and `===...` must be the same length.
The `===...` line in the header must also be exactly the same length.

```suggestion
**numberNative** true/false (boolean) - Whether or not to enable MYSQLI_OPT_INT_AND_FLOAT_NATIVE (``MySQLi`` only).
================ ===========================================================================================================
```

Reply:
Thankfully, I have made changes to this issue.